src/main/java/io/github/patternatlas/api/entities/Pattern.java

```diff
@@ -52,4 +52,6 @@ public class Pattern extends EntityWithURI {
 
     @Column(columnDefinition = "boolean default false")
     private Boolean deploymentModelingBehaviorPattern;
+
+    private String tags;
```

Comment:
is this one tag or multiple? If it is multiple - shouldn't it then be some collection of Strings?

Reply:
We discussed that we want to keep it as "tags" because we might want to store multiple tags as a comma-separated list in this string.